managed/ui/src/components/universes/UniverseForm/ClusterFields.js

```diff
@@ -385,6 +386,15 @@ export default class ClusterFields extends Component {
     let providerSelected = this.state.providerSelected;
     if (isNonEmptyObject(currentFormValues) && isNonEmptyString(currentFormValues.provider)) {
       providerSelected = currentFormValues.provider;
+      if (providerSelected && regions.data.length > 0) {
+        const providerRegions = regions.data &&
+          regions.data.map(function (regionItem) {
+          return { value: regionItem.uuid, label: regionItem.name };
+        });
+        this.setState({
+          defaultRegion: providerRegions[0]
+        });
```

Comment:
No need to put the regions into another array. Since you already checked the length in the if-statement, just access the first value:
```
this.setState({
  defaultRegion: { value: regions.data[0].uuid, label: regions.data[0].name }
})
```

Reply:
I initially thought about that option, but it won't work due to the reason I mentioned here:

https://github.com/yugabyte/yugabyte-db/pull/10028#issuecomment-921414760

@yb-andrew : Please let me know about your thoughts